worlds/stardew_valley/content/mods/archeology.py

```diff
@@ -1,10 +1,19 @@
 from ..game_content import ContentPack
 from ..mod_registry import register_mod_content_pack
+from ...data.game_item import ItemTag, Tag
+from ...data.shop import ShopSource
 from ...data.skill import Skill
 from ...mods.mod_data import ModNames
+from ...strings.book_names import ModBook
+from ...strings.region_names import LogicRegion
 from ...strings.skill_names import ModSkill
 
 register_mod_content_pack(ContentPack(
     ModNames.archaeology,
-    skills=(Skill(name=ModSkill.archaeology, has_mastery=False),)
+    skills=(Skill(name=ModSkill.archaeology, has_mastery=False),),
+    shop_sources={
```

Comment:
Nitpicking, but I think we should try to have each element of content packs set respecting their call order, just like we place the methods in the World in the order they're called by the generation. So basically, shop sources should be defined before skills.